packages/bpk-component-loading-button/src/BpkLoadingButton.js

```diff
@@ -119,6 +140,7 @@ BpkLoadingButton.defaultProps = {
   disabled: false,
   secondary: false,
   destructive: false,
+  large: false,
```

Comment:
Added this prop instead of spreading with `...rest` as its used here: 

https://github.com/Skyscanner/backpack/pull/1942/files#diff-ac38633aeb0e164740ff850d58f35e9dR53

and flow complains that its not a valid prop as its not available in the main props of the component when its used for those functions